scripts/npc/9201131.js

```diff
@@ -0,0 +1,44 @@
+/*
+	Demon's Doorway
+	Treasure, and Crocell the Demon
```

Comment:
You mixed up Bowman and Pirate's quest, but not a big deal since required jobs, target map and NPC ID match up.

Reply:
Comments have been fixed.